### PR TITLE
OAuth accessibility

### PR DESCRIPTION
**Pull request by Astro-Choco**
- Fixed headers and other tagging.
- Moved hyperlinks to the beginning or the end of sentences.
- Changed the file extension to Markdown as most tags in the file are Markdown tags.
- Updated the cross-reference to the file in README.

@sherifzain except the authorization doc, I updated the other docs due to the file extension change to Markdown.

Related JIRA ticket: DEVX-646, 656.657, 658

### CODE REVIEW

File: faq.adoc [hunk 1]
```diff
@@ -96,7 +96,8 @@ Example response
 }
 
 --
-For more information on the filter and the endpoint, see https://github.com/iZettle/api-documentation/blob/master/authorization.adoc[OAuth2 API documentation].
+
+For more information on the filter and the endpoint, see xref:/authorization.md[OAuth] for more info.
```
Comment (AndreaFilyo) [inline faq.adoc:100]: "For more information on the filter and the endpoint, see xref:/authorization.md[OAuth]".

**Review by AndreaFilyo — APPROVED** [review]
Looks good, only one small thing.

File: faq.adoc (hunk 1 above)
Comment (Astro-Choco) [reply faq.adoc:100]: Thanks! Fixed it.

**Review by sherifzain — APPROVED** [review]
LGTM

**Pull request merged**